resources/default_exporters.json

```diff
@@ -398,6 +398,21 @@
       {
         "action": "copy",
         "from": "contact.country",
+        "to": "Land_Name"
```

Comment:
We don't use this variable, but I've found that if I remove the copy on `contact.country`, `var:contact.country[_id]` will be empty for the `load` below. Maybe a weird side-effect of the cache, though I haven't dug too deep.